src/platform/elements/form/extras/file/FileInput.ts

```diff
@@ -40,178 +40,198 @@ const LAYOUT_DEFAULTS = { order: 'default', download: true, labelStyle: 'default
                     <i *ngIf="layoutOptions.draggable" class="bhi-move"></i>
                     <label>{{ file.name | decodeURI }}</label>
                     <div class="actions" [attr.data-automation-id]="'file-actions'" *ngIf="file.loaded">
+                      <div *ngIf="!layoutOptions.customActions">
                         <button *ngIf="layoutOptions.download" type="button" theme="icon" icon="save" (click)="download(file)" [attr.data-automation-id]="'file-download'" tabindex="-1"></button>
                         <button type="button" theme="icon" icon="close" (click)="remove(file)" [attr.data-automation-id]="'file-remove'" tabindex="-1"></button>
+                      </div>
+                      <div *ngIf="layoutOptions.customActions">
```

Comment:
layoutOptions.customActions aren't in the type?

Can we avoid putting more items in this layoutOptions and just do straight @Input() for each. This config object makes it very difficult to use the element over time.